modules/sql-engine/src/integrationTest/sql/set/test_set_distinct.test

```diff
@@ -11,14 +11,52 @@ SELECT 1 UNION DISTINCT SELECT * FROM (values(1),(2),(2),(3),(3),(3),(4)) s(x)
 3
 4
 
-query I
-SELECT * FROM (values(1),(2),(2),(3),(3),(3),(5)) s(x) EXCEPT DISTINCT SELECT * FROM (values (2),(2),(3),(3),(4)) s(x)
+query I rowsort
+SELECT * FROM (values (1),(2),(2),(3),(3),(3),(5)) s(x) EXCEPT DISTINCT SELECT * FROM (values (2),(2),(3),(3),(4)) s(x)
 ----
 1
 5
 
-query I
-SELECT * FROM (values(1),(2),(2),(3),(3),(3),(5)) s(x) INTERSECT DISTINCT SELECT * FROM (values (2),(2),(3),(3),(4)) s(x)
+query I rowsort
+SELECT * FROM (values (1),(2),(2),(3),(3),(3),(5)) s(x) INTERSECT DISTINCT SELECT * FROM (values (2),(2),(3),(3),(4)) s(x)
+----
+2
+3
+
+query I rowsort
+SELECT * FROM (values (1),(2),(2),(3),(3),(3),(5)) s(x) 
+INTERSECT DISTINCT 
+SELECT * FROM (values (2),(2),(3),(3),(4), (7)) s(x) 
+  INTERSECT DISTINCT SELECT * FROM (values (2),(3),(4), (7)) s(x)
 ----
 2
 3
+
+query I
+SELECT * FROM (values (1)) s(x) INTERSECT DISTINCT SELECT * FROM (values (2),(2),(3),(3),(4)) s(x)
```

Comment:
can you append also case with **null** in values

Reply:
Done.